static/sass/_utility_crop.scss

```diff
@@ -0,0 +1,31 @@
+@mixin blog-u-crop {
+  // Crop to 16:9
+  .u-crop--16-9 {
+    position: relative;
+
+    &::before {
+      content: "";
+      display: block;
+      padding-top: 56.25%;
+      width: 100%;
+    }
+
+    // Incase img isn't wrapped in a link
+    > a,
+    > img {
+      bottom: 0;
+      left: 0;
+      overflow: hidden;
+      position: absolute;
+      right: 0;
+      top: 0;
+    }
+
+    > a {
```

Comment:
The cascade will apply both the absolute positioning from above and this flex positioning - I would have thought some strange artifacts/ edge cases could result?

Reply:
Same

Reply:
So, this works fine. I guess it just gets positioned correctly and then handles its own box model with flexbox. No problems I could find here.